tests/marqo_test.py

```diff
@@ -226,6 +233,11 @@ def create_cloud_index(self, index_name, settings_dict=None, **kwargs):
             settings_dict.update({
                 "inference_type": "marqo.CPU", "storage_class": "marqo.basic"
             })
+
+        if len(index_name) > 32:
+            index_name = index_name[len(index_name) - 32: len(index_name)]
+            if index_name[0] == "-":
+                index_name = index_name[1:]
```

Comment:
does this affect any logic that tests for index names starting with a certain pattern?